src/main/java/com/synopsys/integration/jenkins/scan/service/scan/blackduck/BlackDuckParametersService.java

```diff
@@ -62,21 +66,38 @@ public BlackDuck createBlackDuckObject(Map<String, Object> blackDuckParametersMa
                     break;
                 case ApplicationConstants.BLACKDUCK_AUTOMATION_PRCOMMENT_KEY:
                     if (value.equals("true") || value.equals("false")) {
-                        blackDuck.getAutomation().setPrcomment(Boolean.parseBoolean(value));
+                        blackDuck.getAutomation().setPrComment(Boolean.parseBoolean(value));
                     }
                     break;
+                default:
+                    break;
             }
         }
 
         return blackDuck;
     }
 
     public boolean performBlackDuckParameterValidation(Map<String, Object> blackDuckParams) {
-        return blackDuckParams != null
+        boolean isValid =  blackDuckParams != null
                 && Stream.of(ApplicationConstants.BLACKDUCK_URL_KEY, ApplicationConstants.BLACKDUCK_API_TOKEN_KEY)
-                .allMatch(key -> blackDuckParams.containsKey(key)
-                        && blackDuckParams.get(key) != null
-                        && !blackDuckParams.get(key).toString().isEmpty());
+                .allMatch(key -> {
+                    boolean isKeyValid = blackDuckParams.containsKey(key)
+                            && blackDuckParams.get(key) != null
+                            && !blackDuckParams.get(key).toString().isEmpty();
+
+                    if (!isKeyValid) {
+                        listener.getLogger().printf(LogMessages.BLACKDUCK_PARAMETER_VALIDATION_FAILED_FOR_PARAM, key);
```

Comment:
Can we have this logging in single line with all the missed mandatory key names.

Reply:
Updated here - https://github.com/synopsys-sig/synopsys-security-scan-plugin/blob/dev/src/main/java/com/synopsys/integration/jenkins/scan/service/scan/blackduck/BlackDuckParametersService.java#L100

Can you check if this will works or not?